src/deploy/osp_deployer/deployer.py

```diff
@@ -231,7 +231,14 @@ def deploy():
                                     settings.director_install_account_pwd,
                                     cmd)[0]
         logger.debug("=== Overcloud stack state : " + overcloud_status)
-
+        if settings.hpg_enable:
```

Comment:
There is a retreive_nodes_ips method in director.py logging the  deployment related information , those should probably go in there.

Reply:
I think that's a fine idea, but let's change the name of the method to something like "summarize_deploment" or something like that.  I displays information about the overcloud nodes, but also displays endpoints too.

Reply:
@jproque-dell  look at director.py retreive_nodes_ips , that function logs all the deployment related informations ( nodes ips, urls, passwords etc.. ) both to the output and to a separate log file. Those nfv related info should go in there as well, like Chris is suggesting, just rename that method to something more generic.

Reply:
Renamed retreive_node_ips to summarize_deployment as suggested by Chris and included all NFV Features info coming from deployer into this method.